StudCity.Frontend/src/components/Group/GroupCard.jsx

```diff
@@ -1,29 +1,33 @@
 import React from 'react'
+import GridCard from './GridCard'
+import ListCard from './ListCard'
+
+const GroupCard = ({ group, handleClick, viewMode }) => {
+	const stylesContainerCardMode = () => {
```

Comment:
I think will be better do this stylesContainerCardMode as object and

```
const containerCardStyles = {
    grid: "duration-200 hover:scale-90 h-60",
    list: "h-36 mt-5 w-11/12 mx-auto duration-200 hover:scale-105",
    default: "",
  };

// this u can use in rendering body of this component
  const stylesContainerCardMode = containerCardStyles[viewMode];
```

Next switch `renderCardMode` LGTM